pkg/tfvars/ovirt/ovirt.go

```diff
@@ -29,8 +29,8 @@ type config struct {
 	NetworkName             string                   `json:"ovirt_network_name,omitempty"`
 	VNICProfileID           string                   `json:"ovirt_vnic_profile_id,omitempty"`
 	AffinityGroups          []map[string]interface{} `json:"ovirt_affinity_groups,omitempty"`
-	BaseImageName           string                   `json:"openstack_base_image_name,omitempty"`
-	BaseImageLocalFilePath  string                   `json:"openstack_base_image_local_file_path,omitempty"`
+	BaseImageName           string                   `json:"ovirt_base_image_name,omitempty"`
```

Comment:
See my comment above, please consistently rename it everywhere in the oVirt codebase.